support/client/lib/vwf/view/blockly.js

```diff
@@ -451,11 +450,13 @@ define( [ "module", "vwf/view", "jquery", "vwf/model/blockly/JS-Interpreter/acor
     function cleanUpdateToolbox( xml ) {
         // negateOverlap is being use to negate some of the effect of the MARGIN variable
         // in Blockly.createDom_ when deleting blocks over the flyout
-        var negateOverlap = 35;
+        // SJF: Overlap not present in Blockly with categories enabled
+
+        // var negateOverlap = 35;
         Blockly.updateToolbox( xml );
```

Comment:
If the only thing that's executing in this function is `Blockly.updateToolbox( xml )`, does it make more sense to comment out the entire `cleanUpdateToolbox` function, and replace it with `updateToolbox`?